data/config/schema.xml

```diff
@@ -142,7 +142,7 @@ to save the current settings to XML.
 	</entry>
 	<entry name="game/drum_promode" type="bool" value="true">
 		<short>Drums use Pro mode</short>
-		<long>Drums use Pro mode (toms are not cymbals)</long>
+		<long>Drums use Pro mode (toms are not cymbals).</long>
```

Comment:
In English, punctuation goes inside the parentheses.

Reply:
That depends but in this case you're right
Will fix it tomorrow

Reply:
Fixed